jira/utils/__init__.py

```diff
@@ -77,8 +76,10 @@ def threaded_requests(requests):
 
 def json_loads(r):
```

Comment:
@hdost I can split this change out into another pull request if that is desired.